agenda-webapps/src/main/webapp/vue-app/agenda/components/user-setting-agenda/UserSettingAgendaDrawer.vue

```diff
@@ -0,0 +1,110 @@
+          <div class="d-flex flex-column mb-5">
+            <label class="switch-label-text mt-1 text-subtitle-1">{{ $t('agenda.settings.drawer.label.DefaultView') }}:</label>
+            <select v-model="value.agendaDefaultView" class="width-auto my-auto pr-2 subtitle-1 ignore-vuetify-classes d-none d-sm-inline">
+              <option value="day">{{ $t('agenda.label.viewDay') }}</option>
+              <option value="week">{{ $t('agenda.label.viewWeek') }}</option>
+              <option value="month">{{ $t('agenda.label.viewMonth') }}</option>
+            </select>
+          </div>
+          <div class="d-flex flex-column mb-5">
+            <label class="switch-label-text mt-1 text-subtitle-1">{{ $t('agenda.settings.drawer.label.WeekStartOn') }}:</label>
+            <select v-model="value.agendaWeekStartOn" class="width-auto my-auto pr-2 subtitle-1 ignore-vuetify-classes d-none d-sm-inline">
+              <option value="1,2,3,4,5,6,0">{{ $t('agenda.settings.drawer.day.monday') }}</option>
+              <option value="2,3,4,5,6,0,1">{{ $t('agenda.settings.drawer.day.tuesday') }}</option>
+              <option value="3,4,5,6,0,1,2">{{ $t('agenda.settings.drawer.day.wednesday') }}</option>
+              <option value="4,5,6,0,1,2,3">{{ $t('agenda.settings.drawer.day.thursday') }}</option>
+              <option value="5,6,0,1,2,3,4">{{ $t('agenda.settings.drawer.day.friday') }}</option>
+              <option value="6,0,1,2,3,4,5">{{ $t('agenda.settings.drawer.day.saturday') }}</option>
+              <option value="0,1,2,3,4,5,6">{{ $t('agenda.settings.drawer.day.sunday') }}</option>
+            </select>
+          </div>
+
+          <div class="d-flex flex-row mt-5">
+            <label class="switch-label-text mt-1 text-subtitle-1">{{ $t('agenda.settings.drawer.label.showWorkingTime') }}:</label>
+            <v-switch v-model="value.showWorkingTime" class="mt-0 ml-4" />
+          </div>
+          <div class="d-flex flex-row align-baseline">
+            <time-picker v-model="value.workingTimeStart" :disabled="!value.showWorkingTime" />
+            <label class="switch-label-text mx-5 text-subtitle-1" :class="{'disabled': !value.showWorkingTime}">{{ $t('agenda.settings.drawer.label.to') }}</label>
+            <time-picker v-model="value.workingTimeEnd" :disabled="!value.showWorkingTime" />
+          </div>
+        </v-layout>
+      </div>
+    </template>
+    <template slot="footer">
+      <div class="d-flex">
+        <v-spacer />
+        <v-btn
+          class="btn mr-2"
+          @click="close">
+          <template>
+            {{ $t('agenda.settings.drawer.button.cancel') }}
+          </template>
+        </v-btn>
+        <v-btn
+          class="btn btn-primary"
+          @click="save">
+          {{ $t('agenda.settings.drawer.button.save') }}
+        </v-btn>
+      </div>
+    </template>
+  </exo-drawer>
+</template>
+
+<script>
+export default {
+  props: {
+    value: {
```

Comment:
no need to use v-model in this drawer, use just `:settings="settings"`

Reply:
We need the v-model to display the settings when we re-use the component in the global user settings.